### DISCUSSION TITLE
Config to allow magit-merge-into to delete remote branch always

### DISCUSSION THREAD

**Discussion by mkleehammer** [magit/magit#4628] [Ideas]
I'm big fan of merge-into (and the old name ;) but our workflow doesn't use magit-branch-pull-request, so remote branches don't get deleted.

Very often we simply create topic branches, ensure unit tests work, then merge into master.  This seems like a pretty common workflow.

Would it be possible to make merge-into not be dependant on pull-requests?  Perhaps a flag to indicate it should delete the remote branch if not created by magit-branch-pull-request, without the force push either.

Thanks.

**Comment by dkleehammer** [comment]
Removing the branch remotely would work perfectly!  Hope this happens.
